resources/ui-app/package.json

```diff
@@ -12,16 +12,17 @@
     "start": "stripes serve",
     "build": "stripes build --output ./output",
     "test": "stripes test nightmare --run demo --show",
-    "lint": "eslint src"
+    "lint": "eslint ."
   },
   "devDependencies": {
-    "@folio/eslint-config-stripes": "^1.1.0",
-    "@folio/stripes-cli": "^1.2.0",
-    "babel-eslint": "^8.0.3",
-    "eslint": "^4.8.0"
+    "@folio/eslint-config-stripes": "^3.2.1",
+    "@folio/stripes-cli": "^1.4.0",
+    "babel-eslint": "^9.0.0",
+    "eslint": "^5.6.0"
   },
   "dependencies": {
     "@folio/stripes-components": "^2.0.0",
```

Comment:
Would you like to upgrade to stripes-components to 3.x while you're at it?

Reply:
Yes I would!